Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -48,6 +46,8 @@ public struct WorldScaleGeoTrackingSceneView: View {
     @State private var currentLocation: Location?
     /// The valid accuracy threshold for a location in meters.
     private var validAccuracyThreshold = 0.0
+    /// The app's tracking requirements for the coaching overlay to show/dismiss.
+    @State private var goal: ARCoachingOverlayView.Goal
```

Comment:
Instead of using a boolean, we use the goal to let the coaching overlay decide for itself when to show or dismiss. For instance, if the geo-tracking accuracy drops to low or undetermined, the overlay may show again to prompt the user to re-localize the AR view.

Reply:
The comment above is incorrect. For both world tracking and geo tracking configurations, we can use the same `.geoTracking` goal, and the overlay is smart enough to display different prompt.